neo4j-quickstart/pom.xml

```diff
@@ -73,6 +74,58 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <!-- Automatically start Neo4J for integration testing - requires Docker -->
+        <groupId>io.fabric8</groupId>
```

Comment:
Any reason to not use TestContainers as others quickstart (like Kafka)

Reply:
Not really. I thought it'd be better to folow the hibernate quickstarts. I do prefer testcontainers, and didn't noticed it was used. Do you want me to change it?

Reply:
The advantage of test container is that you can run the tests from your IDE or from your terminal.

Reply:
Ok, the changes are done. I don't think the changes can have something to do with the fail on the pipeline. Can we try and run it again?

Reply:
Yes, look at https://github.com/quarkusio/quarkus-quickstarts/blob/development/kafka-quickstart/src/test/java/org/acme/kafka/KafkaResource.java for example.

Reply:
I saw it, and the change is already done in the last commit, I was just asking if there is any other change that needs to be made...

Reply:
ah sorry!